src/fzx/tui/term_app.cpp

```diff
@@ -69,7 +69,7 @@ void TermApp::processTTY()
     } else if (*key == kCtrlU) {
       mLine.clear();
       updateQuery = true;
-    } else if (*key == kCtrlC) {
+    } else if (*key == kCtrlC or *key == kCtrlLBracket) {
```

Comment:
You can add `kEscape` that duplicates the value of `kCtrlLBracket`, should be a little bit more clear. Also I've been using `||` instead of `or`